packages/gradle-plugin/src/main/kotlin/io/realm/kotlin/gradle/RealmCompilerSubplugin.kt

```diff
@@ -50,8 +49,7 @@ class RealmCompilerSubplugin : KotlinCompilerPluginSupportPlugin {
     }
 
     override fun getPluginArtifactForNative(): SubpluginArtifact {
```

Comment:
Add todo to remove this in a separate PR

Reply:
We cannot remove this method as it is part of the interface. It is marked as deprecated though, so I would expect this to be gone in a future Kotlin release.